semgrep/semgrep/metavariable_comparison.py

```diff
@@ -0,0 +1,34 @@
+import collections
+import json
+import subprocess
+import tempfile
+from typing import Union
+
+from semgrep.constants import PLEASE_FILE_ISSUE_TEXT
+from semgrep.constants import SEMGREP_PATH
+from semgrep.error import SemgrepError
+from semgrep.util import sub_check_output
+
+
+def metavariable_comparison(
+    metavariable: str, comparison: str, content: Union[int, float]
+) -> bool:
+    # semgrep-core requires the 'language' key be before the 'code' key,
```

Comment:
I can fix that and sort on the semgrep-core side. This is ugly indeed otherwise to impose an order.
Sorry.

Reply:
No worries! That was going to be one of my v2 feature requests. That and `float` support :)